### PR TITLE
Trying to get around JVM bug

### PR DESCRIPTION
**Pull request by kiiadi**

### CODE REVIEW

File: .github/workflows/mac.yml [hunk 1]
```diff
@@ -17,11 +17,16 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 11
+    - run: curl -OL "https://corretto.aws/downloads/latest/amazon-corretto-11-x64-macos-jdk.tar.gz"
```
Comment (hunterwerlla) [inline .github/workflows/mac.yml:20]: Can you add a comment in the file on why this is here so we can remove it when they upgrade the jvm version?

Comment (kiiadi) [reply .github/workflows/mac.yml:20]: why remove it? I figure we could continue using corretto...

Comment (hunterwerlla) [reply .github/workflows/mac.yml:21]: Speed? But, fair it should be fine either way

Comment (justinmk3) [reply .github/workflows/mac.yml:21]: If not a source comment,  mentioning the actual bug in the commit message is at least worthwhile.

**Comment by abrooksv** [comment]
Do we still need this/ It seems the string that Coretto team gave us to disable C2 is working to prevent segfaults: https://github.com/aws/aws-toolkit-jetbrains/blob/master/jetbrains-core/build.gradle#L48

**Pull request closed without merging**